apps/src/studio/skins.js

```diff
@@ -608,6 +608,86 @@ function loadHoc2015(skin, assetUrl) {
     'rebelpilot':      { frames: 13, width: 100, height: 100, scale: 1,   renderOffset: { x: 0, y: -25}, activity: 'flee',  speed: constants.SpriteSpeed.SLOW, spritesCounterclockwise: true },
   };
 
+  skin.customObstacleZones = {endor : {}, hoth: {}, starship: {}};
+
+  skin.customObstacleZones.endor.blobs = [
+    { minx: 361, maxx: 413, miny: 261, maxy: 344 },
```

Comment:
nit: Why aren't these capitalized `minX` `maxX` `minY` `maxY`?

As-is, I did a double-take.
![image](https://cloud.githubusercontent.com/assets/1615761/11314813/18255846-8f9e-11e5-8fe7-044522faf37d.png)